v63003/u_inref/gtm8732r.csh

```diff
@@ -0,0 +1,58 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Test that a line length greater than 8192 bytes produces a LSEXPECTED warning
+#
+set x = `$ydb_dist/mumps -run ^%XCMD "write 2**31-1"`
+set y = `$ydb_dist/mumps -run ^%XCMD "write 2**31"`
+set rand = `$gtm_tst/com/genrandnumbers.csh 1 0 $x`
+set rand128 = `$gtm_tst/com/genrandnumbers.csh 1 0 128`
+echo '# Creating data base'
+$gtm_tst/com/dbcreate.csh mumps 1>>db.out
+
+echo '# Running CHANGELOG in sec_shell with log interval 0'
```

Comment:
Can log_interval be highlighted in upper case and changelog be made lower case (this test is about LOG_INTERVAL). Also can "in sec_shell" be removed as that is a test system artifact and nothing to do with the MUPIP REPLICATE command that is being tested.